landing/src/index.css

```
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500;700&display=swap');
```

Stylelint (17.11.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500;700&display=swap'" (import-notation)

(import-notation)